geoglows/data.py

```
@@ -1,32 +1,42 @@
 import os
 import warnings
 
+import numpy as np
 import pandas as pd
 import xarray as xr
 
-from ._constants import get_metadata_table_path
+from ._constants import get_metadata_table_path, get_sfdc_table_path, get_saber_assign_table_path, get_fdc_table_path, get_sfdc_zarr_uri, get_transformer_table_uri
```

Ruff (0.8.2)

`._constants.get_sfdc_table_path` imported but unused

Remove unused import

(F401)